### PR TITLE
Updated dependcies

### PR DESCRIPTION
**Pull request by coot**
@mrBliss could you review.  I put `Nothing` in the last argument of `ProtocolRealPBFT` ([here](https://github.com/input-output-hk/cardano-node/pull/75/commits/8c2f5f5a208ed4c5307cd13a889a766ef22e50c9#diff-ba9f0f06843a3a67a21c656d60e82bc8R116)).

### CODE REVIEW

File: cardano-node/Cardano/Node/CLI.hs [hunk 1]
```diff
@@ -113,7 +113,7 @@ fromProtocol RealPBFT =
     case Consensus.runProtocol p of
       Dict -> return $ SomeProtocol p
   where
-    p = ProtocolRealPBFT defaultDemoPBftParams genesisConfig
+    p = ProtocolRealPBFT defaultDemoPBftParams genesisConfig Nothing
```
Comment (mrBliss) [inline cardano-node/Cardano/Node/CLI.hs]: I believe this `Nothing`(the [`PbftLeaderCredentials`](https://github.com/input-output-hk/ouroboros-network/blob/4e5b99098cc4e4bbf400a9a8e98c42dba6ba574e/ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs#L58)) will break the demo for `--real-pbft`.

A proper solution for this is coming in #60: https://github.com/input-output-hk/cardano-node/pull/60/files?file-filters%5B%5D=.hs#diff-ba9f0f06843a3a67a21c656d60e82bc8R153

So I'm not convinced this PR makes sense. We have already broken the demo too many times by accident, I don't think we should start doing it knowingly :slightly_smiling_face:

I would suggest either waiting for #60 to be merged before updating the dependencies *or* lobbying for #60 to incorporate the dependency bumps from this PR.

Comment (coot) [reply cardano-node/Cardano/Node/CLI.hs]: Yes, let's not break the demo. I can wait for #60 first.

Comment (dcoutts) [reply cardano-node/Cardano/Node/CLI.hs]: #60 is now merged.

**Comment by coot** [comment]
The only thing that survived a rebase is removal of `network-mux` from the cabal file; I think it's still worth merging.

**Pull request merged**